README.md

```diff
@@ -0,0 +1,37 @@
+<p>
+    <a href="https://crates.io/crates/near-token"><img src="https://img.shields.io/crates/dv/near-token?style=flat-square&logo=near&label=crates.io" alt="Crates.io (latest)"></a>
+    <a href="https://docs.rs/near-token/latest/near_token"><img src="https://img.shields.io/docsrs/near-token?style=flat-square" alt="Docs.rs"></a>
+    <img src="https://img.shields.io/badge/rustc-1.68%2B-lightgray.svg?style=flat-square" alt="Rust Version">
+</p>
+
+# near-token
+near-token is crate for work with tokens in near-protocol.
```

Comment:
Add a link to https://docs.near.org/concepts/basics/tokens and/or https://wiki.near.org/overview/getting-started#what-is-the-near-token-and-how-can-i-use-it